NanoVNASaver/Hardware/NanoVNA_V2.py

```diff
@@ -223,21 +223,6 @@ def readVersion(self) -> 'Version':
         logger.debug("readVersion: %s", result)
         return result
 
-    def read_board_revision(self) -> 'Version':
```

Comment:
This isn't a duplicate of readVersion.

readVerstion calls CMD FIRMWARE_MAJOR and CMD FIRMWARE_MINOR (0xf3, 0xf4)
read_board_revicsion calls ADDR_DEVICE_VARIANT and ADDR_HARDWARE_REVISION (0xf1, 0xf2)

Reply:
I missed that, thanks. Then you can just add the IOError to read_board_revision() to fix the error, like you suggested below.